AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Campaigns/CampaignDetailQueryHandler.cs

```diff
@@ -82,5 +90,13 @@ public CampaignDetailQueryHandler(AllReadyContext context)
 
             return result;
         }
+
+        private CampaignDetailViewModel.CampaignManagerInviteStatus GetCampaignManagerInviteStatus(CampaignManagerInvite campaignManagerInvite)
+        {
+            if (campaignManagerInvite.IsAccepted) return CampaignDetailViewModel.CampaignManagerInviteStatus.Accepted;
+            else if (campaignManagerInvite.IsPending) return CampaignDetailViewModel.CampaignManagerInviteStatus.Pending;
+            else if (campaignManagerInvite.IsRejected) return CampaignDetailViewModel.CampaignManagerInviteStatus.Rejected;
+            else  return CampaignDetailViewModel.CampaignManagerInviteStatus.Revoked;
+        }
```

Comment:
@stevejgordon thanks for the info/feedback. @mk0sojo, I'd say for now, let's keep the transformations in the CampaignDetailqueryHandler as-is. This PR is already big enough, and I'd like for us to finish it up and get it merged.

I'm going to do some analysis on this and figure out the best way forward. Right now, I'm thinking we can expose the enum on ManagerInvite, but we'll need to keep the dates when the different statuses were set for what I'm assuming is a history/audit trail purpose.